Gemfile

```diff
@@ -1,7 +1,7 @@
 source "http://rubygems.org"
 # Add dependencies required to use your gem here.
 
-gem "httparty", "~> 0.7"
+gem "httparty", "~> 0.10.0"
```

Comment:
see https://github.com/flowdock/capistrano-node/commit/da3d8c5d9a40544c43e733c65b3e10d48d28e034 for tips on pinning rake to avoid the travis failure